homework-submission/week1-project/src/Todo app/TodoApp.js

```diff
@@ -0,0 +1,129 @@
+import {observable, computed, action, runInAction} from 'mobx';
+
+class TodoApp{
+    @observable todos = {
+        data: [],
+        status:  'loading'
```

Comment:
According to the application logic, perhaps you may have another look to the state status here, I meant `pending` 🤔